ostree/README.md

```diff
@@ -7,27 +7,24 @@ There are two ways of creating an update image:
 
 These artifacts are produced during the Yocto build.
 
-Scripts are provided for both of these, and instructions for running directly or with Docker. The Docker method has been tested running on an Ubuntu 16.04 machine only.
-
 # Creating an upgrade tarball between wic images
 
 The createOSTreeUpgrade.sh script can be used to create a field upgrade tarball.
 
 ```
-> ./createOSTreeUpgrade.sh old-wic-file new-wic-file [upgrade-tag]
+> ./createOSTreeUpgrade.sh old-wic-file new-wic-file output-file
 ```
 
 If using the Docker container use:
 
 ```
 docker build --no-cache -f Docker/Dockerfile --label ostree-delta  --tag ${USER}/ostree-delta:latest .
-docker run --rm -v old_wic_file:/old_wic.wic -v new_wic_file:new_wic.wic -v /dev:/dev -v ${PWD}:/ws -w /ws --privileged ${USER}/ostree-delta:latest ./createOSTreeUpgrade.sh /old_wic.wic /new_wic.wic [upgrade-tag]
+docker run --rm -v old-wic-file:/old_wic.wic -v new-wic-file:/new_wic.wic -v /dev:/dev -v ${PWD}:/ws -w /ws --privileged ${USER}/ostree-delta:latest ./createOSTreeUpgrade.sh /old_wic.wic /new_wic.wic output-file
 ```
 
 Notes:
-  1. The output is a gzipped tarball.
-  1. old-wic-file and new-wic-file are the .wic images produced by Yocto build
-  1. output is either data.tar.gz or upgrade-tag-data.tar.gz
+  1. old-wic-file and new-wic-file are the absolute paths to the .wic images produced by Yocto build
+  1. The output-file is a gzipped tarball.
```

Comment:
and exact name of it is?

Reply:
The filename is whatever the user specifies. It is the last parameter in the `docker run` command line.